org.sdrangel.SDRangel.yaml

```diff
@@ -0,0 +1,402 @@
+        sha256: 1ec1cba65f9f20fe5a41fda1586e01c70ea0c9a6d7b67c9e13edf0cfe2239277
+
+  - name: libpostproc
+    config-opts:
+      - --disable-debug
+      - --disable-doc
+      - --disable-static
+      - --enable-shared
+      - --enable-gpl
+      - --disable-libvpx
+      - --disable-ffplay
+      - --disable-ffprobe
+      - --disable-everything
+      - --enable-postproc
+    sources:
+      - type: archive
+        url: https://ffmpeg.org/releases/ffmpeg-5.0.tar.xz
+        sha256: 51e919f7d205062c0fd4fae6243a84850391115104ccf1efc451733bc0ac7298
+
+  - name: libaio
+    no-autogen: true
+    make-install-args:
+      - prefix=${FLATPAK_DEST}
+    sources:
+      - type: archive
+        url: https://pagure.io/libaio/archive/libaio-0.3.113/libaio-libaio-0.3.113.tar.gz
+        sha256: 716c7059703247344eb066b54ecbc3ca2134f0103307192e6c2b7dab5f9528ab
+
+  - name: cm256cc
+    buildsystem: cmake-ninja
+    config-opts:
+      - -Wno-dev
+    sources:
+      - type: archive
+        url: https://github.com/f4exb/cm256cc/archive/refs/tags/v1.1.0.tar.gz
+        sha256: 5e6643b7fe7f2dcebf5f602451a77d82959732dbe83bda5029958f9cda6ddac8
+
+  - name: mbelib
+    buildsystem: cmake-ninja
+    config-opts:
+      - -Wno-dev
+    sources:
+      - type: archive
+        url: https://github.com/szechyjs/mbelib/archive/refs/tags/v1.3.0.tar.gz
+        sha256: 5a2d5ca37cef3b6deddd5ce8c73918f27936c50eb0e63b27e4b4fc493310518d
+
+  - name: serialdv
+    buildsystem: cmake-ninja
+    config-opts:
+      - -Wno-dev
+    sources:
+      - type: archive
+        url: https://github.com/f4exb/serialDV/archive/refs/tags/v1.1.4.tar.gz
+        sha256: 0764701ac7b52ab4e7db26d91b0c0d5206377574a8ce4d705c9ec99b8f85ef79
+
+  - name: dsdcc
+    buildsystem: cmake-ninja
+    config-opts:
+      - -Wno-dev
+      - -DUSE_MBELIB=ON
+    sources:
+      - type: archive
+        url: https://github.com/f4exb/dsdcc/archive/refs/tags/v1.9.3.tar.gz
+        sha256: c87f7db419c12ca43f2af58feec4efcb4a8164b32e51474ac1b797338ce5695a
+
+  - name: codec2
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -Wno-dev
+    sources:
+      - type: archive
+        url: https://github.com/drowe67/codec2/archive/refs/tags/v1.0.5.tar.gz
+        sha256: cd9a065dd1c3477f6172a0156294f767688847e4d170103d1f08b3a075f82826
+
+  - name: faad2
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/faac/files/faad2-src/faad2-2.8.0/faad2-2.8.8.tar.gz
+        sha256: 985c3fadb9789d2815e50f4ff714511c79c2710ac27a4aaaf5c0c2662141426d
+
+  - name: libdab
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/srcejon/dab-cmdline.git
+        commit: 4677a078f05642296aaf82bb744e88bb6427ff28
+    build-commands:
+      - cd library && mkdir build && cd build && cmake -DCMAKE_INSTALL_PREFIX=/app .. && make -j $(nproc) && make install
+
+  - name: hidapi
+    buildsystem: cmake-ninja
+    cleanup:
+      - /lib/*.la
```

Comment:
```suggestion
      - '*.la'
      - '*.a'
```

Reply:
Hmm, this was taken from another Flatpak manifest and I think it should be correct as it is. Or am I missing something?

Reply:
"another flaptak manifest" doesn't mean correct, review is only at submission time.

If there are .la there are .a usually

Reply:
Ok, fair point. Thanks!

Reply:
Anyway, is it ok to remove the `/lib/` prefix?

Reply:
I have applied your suggested changes. Hopefully they are fine. :-) Thanks!